Detectors/EMCAL/workflow/src/StandaloneAODProducerSpec.cxx

```diff
@@ -0,0 +1,173 @@
+  auto& caloCellsBuilder = pc.outputs().make<TableBuilder>(Output{"AOD", "CALO"});
+  auto& caloCellsTRGTableBuilder = pc.outputs().make<TableBuilder>(Output{"AOD", "CALOTRIGGER"});
+
+  auto bcCursor = bcBuilder.cursor<o2::aod::BCs>();
+  auto collisionsCursor = collisionsBuilder.cursor<o2::aod::Collisions>();
+  auto caloCellsCursor = caloCellsBuilder.cursor<o2::aod::Calos>();
+  auto caloCellsTRGTableCursor = caloCellsTRGTableBuilder.cursor<o2::aod::CaloTriggers>();
+
+  // build event for easier handling
+  mCaloEventHandler->reset();
+  mCaloEventHandler->setCellData(cellsIn, triggersIn);
+
+  std::vector<int64_t> bcMap;
+  // loop over events
+  for (int iev = 0; iev < mCaloEventHandler->getNumberOfEvents(); iev++) {
+    o2::emcal::EventData inputEvent = mCaloEventHandler->buildEvent(iev);
+    auto cellsInEvent = inputEvent.mCells;                  // get cells belonging to current event
+    auto interactionRecord = inputEvent.mInteractionRecord; // get interaction records belonging to current event
+
+    LOG(INFO) << "Found " << cellsInEvent.size() << " cells in event";
+
+    auto bcID = interactionRecord.toLong();
+    bcMap.emplace_back(bcID);
+    for (auto& cell : cellsInEvent) {
+
+      // fill table
+      caloCellsCursor(0,
+                      bcID,
+                      cell.getTower(),
+                      o2::math_utils::detail::truncateFloatFraction(cell.getAmplitude(), mCaloAmp),
+                      o2::math_utils::detail::truncateFloatFraction(cell.getTimeStamp(), mCaloTime),
+                      cell.getType(),
+                      1); // hard coded for emcal (-1 would be undefined, 0 phos)
+    }                     // end of cell loop
+
+    // filled only with BCID, rest dummy for no2
+    caloCellsTRGTableCursor(0,
+                            bcID,
+                            0,  // fastOrAbsId (dummy value)
+                            0., // lnAmplitude (dummy value)
+                            0,  // triggerBits (dummy value)
+                            1); // caloType (dummy value)
+
+    // fill collision cursor
+    collisionsCursor(0,
+                  bcID,
+                  0., // X-Pos dummy value
+                  0., // Y Pos
+                  0., // Z Pos
+                  0, // cov 0
+                  0, // cov 1
+                  0, // cov 2
+                  0, // cov 3
+                  0, // cov 4
+                  0, // cov 5
+                  0, // vertex bit field for flags
+                  0, // chi2
+                  0, // ncontributors
+                  0, // rel interaction time
+                  0);// vertex time stamp
+  }                             // end of event loop
+  // std::cout << "Finished cell loop" << std::endl;
+
+  std::sort(bcMap.begin(), bcMap.end(), std::less<int64_t>());
+  uint64_t triggerMask = 1;
+  for (auto& bcID : bcMap) {
+    bcCursor(0,
+             runNumber,
+             bcID,
+             triggerMask);
+  }
```

Comment:
This should go directly in the event loop. Instead of filling 0 in the index column we use the counter value used in the other tables.